roles/atomic_installation_verify/tasks/main.yml

```diff
@@ -22,17 +22,29 @@
     cockpit_cname: "docker.io/cockpit/ws:latest"
   when: ('CentOS' in ansible_distribution) or ansible_distribution == "Fedora"
 
+- name: Pull the cockpit container for inspection
+  command: docker pull {{ cockpit_cname }}
+
+- name: Get the version of the cockpit-ws rpm in the container
+  command: docker run --rm {{ cockpit_cname }} rpm --queryformat '%{VERSION}\n' -q cockpit-ws
```

Comment:
Yuck. This would be better done by just embedding the version in a label instead.

Actually, that's the case in the RHEL version of the container (it has a `version` label). So maybe cockpit just has to upstream that bit.